pkg/kncloudevents/event_dispatcher.go

```diff
@@ -66,8 +65,22 @@ type DispatchInfo struct {
 	Scheme         string
 }
 
+type EventFormat string
+
+const (
+	Binary EventFormat = "binary"
+	Json   EventFormat = "json"
+)
```

Comment:
Could you re-use the values from https://github.com/knative/eventing/blob/c3baeda069fc5ce82f15d9f1a8a1029f67c34304/pkg/apis/duck/v1/delivery_types.go#L168-L174 instead of creating new constants here?

Reply:
This will also help with calling the library, as there won't need to be any type casts from the type on the delivery struct to the type accepted here